### PR TITLE
Don't repeat names in module and types

### PR DESCRIPTION
**Pull request by klochowicz**
Same refactor as recently on the maker side.

### CODE REVIEW

File: daemon/src/routes_taker.rs
Comment (thomaseizinger) [inline daemon/src/routes_taker.rs]: Can we refer to this via the module name please?
```suggestion
    cfd_actor_inbox: &State<Address<taker_cfd::Actor>>,
```

File: daemon/src/taker_inc_message_actor.rs
Comment (thomaseizinger) [inline daemon/src/taker_inc_message_actor.rs]: Same here.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]
Thanks!

One nit.

File: daemon/src/routes_taker.rs
Comment (klochowicz) [reply daemon/src/routes_taker.rs]: sure!

**Comment by klochowicz** [comment]
@thomaseizinger I've applied the suggestion locally and force-pushed the fixes. How to tell github UI that I've done the requested changes? 😕

**Comment by thomaseizinger** [comment]
> @thomaseizinger I've applied the suggestion locally and force-pushed the fixes. How to tell github UI that I've done the requested changes? confused

I shows them as (Outdated) now plus I got a notification that you pushed something new. If you want to showcase the diff, you could push extra `fixup!` commits and once you get another approval, rebase, force-push and merge.

File: daemon/src/taker_inc_message_actor.rs
Comment (thomaseizinger) [inline daemon/src/taker_inc_message_actor.rs:3]: What I specifically like about this pattern of referring to types via their parent module is that it results in relatively clean and short imports!

**Review by thomaseizinger — APPROVED** [review]

File: daemon/src/taker_inc_message_actor.rs
Comment (klochowicz) [reply daemon/src/taker_inc_message_actor.rs:3]: yes, and indeed not losing it can leave out important context when reading a line - ie. which actor is it ;D

It would be cool to lint that someday indeed.

**Comment by klochowicz** [comment]
> > @thomaseizinger I've applied the suggestion locally and force-pushed the fixes. How to tell github UI that I've done the requested changes? confused
> 
> I shows them as (Outdated) now plus I got a notification that you pushed something new. If you want to showcase the diff, you could push extra `fixup!` commits and once you get another approval, rebase, force-push and merge.



> > @thomaseizinger I've applied the suggestion locally and force-pushed the fixes. How to tell github UI that I've done the requested changes? confused
> 
> I shows them as (Outdated) now plus I got a notification that you pushed something new. If you want to showcase the diff, you could push extra `fixup!` commits and once you get another approval, rebase, force-push and merge.

thanks, I'll start doing that, that should make it easier for reviewers! I was used to a reviewboard workflow, when you get the diffs precomputed from patches you submit...

**Pull request merged**
